### PR TITLE
Carve out IEC 60870-5-104 from Eclipse NeoSCADA

### PR DESCRIPTION
**Pull request by philschaller**
- Move source code from Eclipse NeoSCADA to Eclipse OneOFour without changing the source code
- Update from EPL v1.0 to EPL v2.0
- Update to standard Maven structure
- Update dependencies

### CODE REVIEW

File: org.eclipse.oneofour.client.data.testing/pom.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>org.eclipse.oneofour-parent</artifactId>
+        <groupId>org.eclipse.oneofour</groupId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <name>/org.eclipse.oneofour.client.data.testing</name>
```
Comment (ctron) [inline org.eclipse.oneofour.client.data.testing/pom.xml]: The leading slash looks strange to me. Is that intentional?

**Comment by ctron** [comment]
Awesome, looks good to me. There is minor thing with a leading slash in a module's name.

I noticed that you are no longer creating an OSGi bundle. There is a simple way to do this, without P2 and Tycho. I will create a PR, when I find some time, and you can decide if you want to merge it. I think it would make it easier for people to migrate is OSGi is still an option.

File: org.eclipse.oneofour.client.data.testing/pom.xml (hunk 1 above)
Comment (philschaller) [reply org.eclipse.oneofour.client.data.testing/pom.xml]: The slash and the rather unconventional naming is a remnant from the NeoSCADA project.

**Comment by philschaller** [comment]
I would be happy to merge your OSGi related patch.

File: org.eclipse.oneofour.client.data.testing/pom.xml (hunk 1 above)
Comment (ctron) [reply org.eclipse.oneofour.client.data.testing/pom.xml]: It is? Please get rid of it :grin: Unless you want to keep it.

**Pull request merged**